README.md

```diff
@@ -80,6 +124,7 @@ Support for colocating your styles with your React component.
   the unit is. See
   [CSSProperty.js](https://github.com/facebook/react/blob/master/src/renderers/dom/shared/CSSProperty.js)
   in React.
+- Consider removing !important from everything.
```

Comment:
:+1: having everything be `!important` is pretty annoying with dynamic styles, because if you want a plain `style={...}` style to overwrite an aphrodite style then you need to append `" !important"` to it.